src/OrchardCore.Modules/OrchardCore.Email/Services/SmtpService.cs

```diff
@@ -128,6 +128,23 @@ private MimeMessage FromMailMessage(MailMessage message)
                 body.TextBody = message.Body;
             }
 
+            if (message.Attachments != null)
+            {
+                foreach (var attachment in message.Attachments)
+                {
+                    if (attachment.Stream == null)
+                    {
+                        // Gets the file from the File System. The filename must be the system path to the file. "App_Data/tenants.json" works.
```

Comment:
I don't think we should support this. Too much security risk. What if you email the `appsettings.json` file with the sql server passwords in it?

At some point, someone will want to add ui to this, and then it will be compromised.

I think setting via a `Stream` only is fine. 

It then becomes the callers responsibility to resolve a stream correctly, which would mean when a workflow feature gets added, that feature is responsible for delivering a safe stream.

Should it also have a max size setting?
Probably, but you might have to read the entire stream to get the length.
Most mail providers will set their own limit, so possibly can be managed there.